examples/utils/example_node.rs

```diff
@@ -81,7 +81,7 @@ impl ExampleNode {
                 }
                 Event::RestartRequired => {
                     info!("{} Received RestartRequired event", self.get_debug_name());
-                    self.node = unwrap_result!(Node::builder().create(self.sender.clone()));
+                    let new_node = unwrap!(Node::builder().create(self.sender.clone()));
```

Comment:
Should probably still be `self.node =`.

This file has changed considerably on master and it shouldn't be overwritten with the version from the stable branch. Just change all the `unwrap_result!` and `unwrap_option!` to `unwrap!`.